j2cl-archetypes/README.md

```diff
@@ -0,0 +1,41 @@
+# J2CL Maven Project Archetypes
+
+All archetypes take the usual `artifactId`, `groupId`, `version`, and `package` and also `module` to
+use for naming some classes in the generated projects.
+
+To create a project interactively, use:
```

Comment:
need to grab the "Tip:" about downloading the archetype before trying to generate

Reply:
@niloc132 

```
mvn dependency:get -Dartifact=com.vertispan.j2cl:j2cl-maven-plugin:0.19-SNAPSHOT -DremoteRepositories=sonatype-snapshots::::https://repo.vertispan.com/j2cl

```

this one ?

Reply:
Yep, will grab the text about it too